actions/recipe_action.go

```diff
@@ -1,7 +1,21 @@
 /*
 Recipe Action
 
-Include a recipe.
+This action includes the recipe actions at the given path, and can optionally
+overrides or adds new template variables.
+
+To allow recipes to operate both stand-alone and when included, template
+variable "included_recipe" is added and set to "true" to indicate that the
+recipe is part of a longer chain.
+This allows a recipe to skip the unpack step if it's not the first to be
+executed or to optionally skip a pack action when it's not the last.
```

Comment:
I'm not clear on how this is meant to work from this documentation? This seems to imply that a recipe can know it's "not the first" or if it's "not the last" when all it gets is a boolean saying it was included.

You probably just want to indicate it's up to the recipe writes on how to use this (I keep going back and forward, but i think in my recipe collection i wouldn't end up using this one but be explicit per recipe in enabling/disabling optional steps)

Reply:
```
To allow recipes to operate both stand-alone and when included, template
variable "included_recipe" is added and set to "true" to indicate that the
recipe is part of a longer chain.
This allows a recipe to skip or add some actions, if it's not the first to be
executed, by surrounding them with "included_recipe" test. For example, the
pack action of an included recipe can be skipped when its build result is
directly used by its parent recipe.
```
Do you think this is clearer?

Reply:
No because it doesn't address the core of my complaint.  You're giving examples that cannot be implemented with the provided extra variable as said variable doesn't contain the required information.